app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -73,6 +78,15 @@ protected void log(int priority, String tag, String message, Throwable throwable
         daoSession = new CustomDaoMaster(db).newSession();
 
         VersionHelper.updateAppVersion(getApplicationContext());
+
+        /**
```

Comment:
Move this comment inside the `if` statement (to make it more readable when we add more version upgrades in the future.